app/tests/test.py

```diff
@@ -3,11 +3,15 @@
 import time
 import os
 
+is_travis = 'TRAVIS' in os.environ
 class BadgeyayTest(unittest.TestCase):
 
 	@classmethod
 	def setUpClass(cls):
-		cls.driver = webdriver.PhantomJS()
+		if is_travis:
+			cls.driver = webdriver.Firefox()

```

Comment:
`If is_travis == true`
then it should be using phantom JS.
I think the conditions are reversed unless I am missing something.

Reply:
Yes. I too think this is a mistake 👍

Reply:
@jajodiaraghav @srungta It's correct 😄 
Please see [this](https://stackoverflow.com/questions/2020598/in-python-how-should-i-test-if-a-variable-is-none-true-or-false).
It's just i have to swap the statements, as if travis environment is true than phantomJS should execute else Firefox driver should execute